ner_constants.py

```diff
@@ -57,5 +57,8 @@
 PARAMETER_MAX_DIGITS = 'max_number_digits'
 PARAMETER_NUMBER_UNIT_TYPE = 'unit_type'
 
+# Prior detection results from CRF models.
+PARAMETER_PRIOR_CRF_RESULTS = "free_text_detection_results"
```

Comment:
use a same naming convention for CRF entities

Reply:
Done.